cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -220,6 +220,30 @@ def test_operations_suspended_vm(small_test_vm, provider, soft_assert):
 
 
 @pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider))
+def test_operations_powered_off_vm(small_test_vm, provider, soft_assert):
+    # Make sure the VM is off
+    small_test_vm.power_control_from_cfme(option=small_test_vm.POWER_OFF, cancel=False)
+    small_test_vm.wait_for_vm_state_change(desired_state=small_test_vm.STATE_OFF)
+    # Create first snapshot
+    snapshot1 = new_snapshot(small_test_vm)
```

Comment:
question for second level review: should we consider having a helper for the creation of a snapshot as a followup with a more nice api as this test needs to do quite some plumbing due to low level of the api provided

Reply:
Shouldn't this be resolved with collections? You should then be able to do something like `vm.snapshots.create(...)` ...

Reply:
@mfalesni perfect point, thanks 👍